src/mmds/src/lib.rs

```diff
@@ -22,7 +22,7 @@ lazy_static! {
     // A static reference to a global Mmds instance. We currently use this for ease of access during
     // prototyping. We'll consider something like passing Arc<Mutex<Mmds>> references to the
     // appropriate threads in the future.
-    pub static ref MMDS: Arc<Mutex<Mmds>> = Arc::new(Mutex::new(Mmds::default()));
+    pub static ref MMDS: Arc<Mutex<Mmds>> = Arc::new(Mutex::new(Mmds::new().expect("Unable to create MMDS.")));
```

Comment:
It makes sense to have the `MMDS` instance reference to be an `Option` in order to make sure customers that do not care about MMDS are not affected by MMDS not being successfully created. I logged a warning in case MMDS creation fails. Let me know what you think about this.

It is true that we do not have a way to start without MMDS at the moment - the instance is always created, even if there are no net devices configured to accept mmds requests.